lib/exercism/team_stream.rb

```diff
@@ -8,11 +8,12 @@
 class TeamStream
   attr_reader :team, :viewer_id, :page, :track_id, :user_id, :username, :slug
   attr_accessor :per_page
-  def initialize(team, viewer_id, page=1)
+
+  def initialize(team, viewer_id, page=1, per_page=50)
```

Comment:
This is not going to get the default when you pass params['per_page'] with nil. I would suggest adding the `|| 50` to the call in the route file.